### ISSUE TITLE
sdap code can mark the whole sssd_be offline

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2979]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2976
- Created at 2016-03-16 23:01:05 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed at 2017-12-15 20:06:40 as worksforme
- Assigned to [fidencio](https://pagure.io/user/fidencio)
---

There is a case where, during connection, a low-lever sdap function can mark the whole back end unconditionally as offline.

We might want to instead try to infer the domain from the sdap_domain structure to be able to only disable subdomain instead of marking the whole domain as offline.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-03-29 11:59:11](https://pagure.io/SSSD/sssd/issue/2976#231104)**

Moving to 1.13.4 as agreed on Mar-17 triage.

blockedby:  => 
blocking:  => 
changelog:  => 
coverity:  => 
design:  => 
design_review:  => 0
feature_milestone:  => 
fedora_test_page:  => 
mark: no => 0
milestone: NEEDS_TRIAGE => SSSD 1.13.4
review: True => 0
selected:  => 
testsupdated:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-04-14 17:01:33](https://pagure.io/SSSD/sssd/issue/2976#231105)**

Fields changed

milestone: SSSD 1.13.4 => SSSD 1.13.5

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-10-05 22:14:57](https://pagure.io/SSSD/sssd/issue/2976#231106)**

Nobody touched this ticket for half a year. I suggest to move it to 1.15 Alpha (through triage) and then decide if it's something to fix in the stable branch as well. One goal of the refactoring might be to see if the situation can be improved even w/o many code changes.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-10-05 22:22:08](https://pagure.io/SSSD/sssd/issue/2976#231107)**

Fields changed

milestone: SSSD 1.13.5 => NEEDS_TRIAGE

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-10-07 10:54:41](https://pagure.io/SSSD/sssd/issue/2976#231108)**

This ticket depends on ticket #1507. We need to first take a look at the sdap code and only once it's documented and tested, we can decide if it's possible to fix this bug without 

blockedby:  => 1507

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-10-12 21:45:09](https://pagure.io/SSSD/sssd/issue/2976#231109)**

Moving to the same milestone as #1507 in the meantime.

milestone: NEEDS_TRIAGE => SSSD 1.15 Alpha

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-11-14 12:04:34](https://pagure.io/SSSD/sssd/issue/2976#231110)**

Linked to Bugzilla bug: https://bugzilla.redhat.com/show_bug.cgi?id=1301740 (*Red Hat Enterprise Linux 7*)

rhbz: [https://bugzilla.redhat.com/show_bug.cgi?id=1316713 1316713] => [https://bugzilla.redhat.com/show_bug.cgi?id=1316713 1316713], [https://bugzilla.redhat.com/show_bug.cgi?id=1301740 1301740]

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-01-25 12:15:58](https://pagure.io/SSSD/sssd/issue/2976#231111)**

Fields changed

milestone: SSSD 1.15.0 => SSSD 1.15 Beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-06 15:40:33](https://pagure.io/SSSD/sssd/issue/2976#231112)**

Fields changed

milestone: SSSD 1.15.3 => SSSD 1.15.2

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:43:11](https://pagure.io/SSSD/sssd/issue/2976#231113)**

**Metadata Update from @jhrozek**:
- Issue marked as depending on: #1507
- Issue set to the milestone: SSSD 1.15.2

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-03-03 14:06:25](https://pagure.io/SSSD/sssd/issue/2976#429279)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset
- Custom field mark reset
- Custom field patch reset
- Custom field review reset
- Custom field sensitive reset
- Custom field testsupdated reset
- Issue close_status updated to: None
- Issue set to the milestone: SSSD 1.15.3 (was: SSSD 1.15.2)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-03-03 14:15:21](https://pagure.io/SSSD/sssd/issue/2976#429285)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset
- Custom field mark reset
- Custom field patch reset
- Custom field review reset
- Custom field sensitive reset
- Custom field testsupdated reset

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-03-15 11:46:48](https://pagure.io/SSSD/sssd/issue/2976#431455)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset
- Custom field mark reset
- Custom field patch reset
- Custom field review reset
- Custom field sensitive reset
- Custom field testsupdated reset
- Issue set to the milestone: SSSD 1.15.4 (was: SSSD 1.15.3)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-08-10 14:07:34](https://pagure.io/SSSD/sssd/issue/2976#456611)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue set to the milestone: SSSD 1.16.0 (was: SSSD 1.15.4)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-08-18 09:44:44](https://pagure.io/SSSD/sssd/issue/2976#458579)**

**Metadata Update from @fidencio**:
- Issue assigned to fidencio

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-10-19 20:22:34](https://pagure.io/SSSD/sssd/issue/2976#473610)**

Since we are required to release a new upstream tarball no later than Friday Oct-20, I'm moving tickets that will not be closed by that date to the next milestone, 1.16.1

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-10-19 20:22:35](https://pagure.io/SSSD/sssd/issue/2976#473611)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue set to the milestone: SSSD 1.16.1 (was: SSSD 1.16.0)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-10-19 20:29:06](https://pagure.io/SSSD/sssd/issue/2976#473625)**

@fidencio btw shouldn't this ticket be just closed, based on your testing?

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-10-19 20:29:08](https://pagure.io/SSSD/sssd/issue/2976#473626)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-12-15 20:02:43](https://pagure.io/SSSD/sssd/issue/2976#484944)**

@jhrozek, please, let's close the ticket.

I didn't find any obvious place where it can actually happen and all patches related to this are already pushed (were already pushed for a long time).

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-12-15 20:02:43](https://pagure.io/SSSD/sssd/issue/2976#484945)**

**Metadata Update from @fidencio**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-12-15 20:03:06](https://pagure.io/SSSD/sssd/issue/2976#484947)**

After investigation, the bug should no longer be present in 1.16.1. If you still see the issue in 1.16.1, please open a new ticket.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-12-15 20:03:06](https://pagure.io/SSSD/sssd/issue/2976#484948)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-12-15 20:06:44](https://pagure.io/SSSD/sssd/issue/2976#484950)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue close_status updated to: worksforme
- Issue status updated to: Closed (was: Open)
